src/OrchardCore/OrchardCore.DisplayManagement.Liquid/Tags/ShapeRemovePropertyTag.cs

```diff
@@ -0,0 +1,26 @@
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Fluid;
+using Fluid.Ast;
+using OrchardCore.Liquid.Ast;
+using OrchardCore.Mvc.Utilities;
+
+namespace OrchardCore.DisplayManagement.Liquid.Tags
+{
+    public class ShapeRemovePropertyTag : ExpressionArgumentsTag
+    {
+        public override async ValueTask<Completion> WriteToAsync(TextWriter writer, TextEncoder encoder, TemplateContext context, Expression expression, FilterArgument[] args)
+        {
+            var objectValue = (await expression.EvaluateAsync(context)).ToObjectValue();
+
+            if (objectValue is IShape shape)
+            {
+                var arguments = (FilterArguments)(await new ArgumentsExpression(args).EvaluateAsync(context)).ToObjectValue();
+                shape.Properties.Remove(arguments["property"].Or(arguments.At(0)).ToStringValue().ToPascalCaseUnderscore());
```

Comment:
Does it fail if the property doesn't exist? If so then it should be checked before.

Reply:
I will test this.

Reply:
I tested it, no exceptions thrown if the property does not exist. It only 💥 when you don't pass any parameters.

Reply:
As per: https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.remove?view=netframework-4.8 

> 
> Returns
> Boolean
> true if the element is successfully found and removed; otherwise, false. This method returns false if key is not found in the Dictionary<TKey,TValue>.